content/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry.md

```diff
@@ -164,6 +164,53 @@ subprojects {
    $ gradle publish
   ```
 
+### Using a package
+
+You can use a package by adding the package as a dependency and the repository to your project. For more information, see "[Declaring dependencies](https://docs.gradle.org/current/userguide/declaring_dependencies.html)" in the Gradle documentation.
```

Comment:
What do you think of this wording change? Also, should we say that the package must be available on GitHub Packages?

```suggestion
To use a package from {% data variables.product.prodname_registry %} without installing the package, add the package as a dependency and add the repository to your project. For more information, see "[Declaring dependencies](https://docs.gradle.org/current/userguide/declaring_dependencies.html)" in the Gradle documentation.
```

Reply:
I love it, thanks! And good question! I'm going to add a clarification. Please, let me know your feedback.